### PR TITLE
Feature/channel list header

### PR DESCRIPTION
**Pull request by xsahil03x**
# Submit a pull request

## CLA

- [x] I have signed the [Stream CLA](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) (required).
- [x] The code changes follow best practices
- [x] Code changes are tested (add some information if not applicable)

### CODE REVIEW

File: lib/src/channel_list_header.dart [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+    final _client = client ?? StreamChat.of(context).client;
+    final user = _client.state.user;
+    return AppBar(
+      brightness: Theme.of(context).brightness,
+      elevation: 1,
+      backgroundColor:
+          StreamChatTheme.of(context).channelTheme.channelHeaderTheme.color,
+      centerTitle: true,
+      leading: Center(
+        child: UserAvatar(
```
Comment (imtoori) [inline lib/src/channel_list_header.dart:81]: I'd add 
```      
showOnlineStatus: false,
```

File: lib/src/channel_list_header.dart [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+                    break;
+                  case ConnectionStatus.connecting:
+                    color = Colors.grey;
+                    break;
+                  case ConnectionStatus.disconnected:
+                    color = Colors.grey;
+                    break;
+                }
+                return Icon(
+                  StreamIcons.pen_write,
```
Comment (imtoori) [inline lib/src/channel_list_header.dart:110]: this icon is wrong
if you don't find it download it from figma

**Review by imtoori — CHANGES_REQUESTED** [review]
added two comments

File: lib/src/channel_list_header.dart (hunk 1 above)
Comment (xsahil03x) [reply lib/src/channel_list_header.dart:81]: Done 👍

**Pull request merged**